packages/date-picker/src/DatePicker/DatePickerMenu/DatePickerMenuSelect/DatePickerMenuSelectMonth.tsx

```diff
@@ -43,7 +43,9 @@ export const DatePickerMenuSelectMonth = ({
   return (
     <Select
       {...selectElementProps}
-      aria-label="Select month"
+      aria-label={`Select month - ${
+        monthOptions[month.getUTCMonth()].long
+      } selected`}
```

Comment:
This is probably a wider issue with Select, right?

Reply:
Yeah, if you're using a select without a label and need to use aria-label then it becomes an issue.

Reply:
I can create a ticket for this